### DISCUSSION TITLE
latest node version possible with v1?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#13580] [CLI - PWA mode] [answered]
Is this documentation still valid? https://v1.quasar.dev/quasar-cli/installation

Although it says no higher than v12 I have been using node v14 for quite some time with (seemingly) no issues, and was wanting to use later versions if possible.

**Comment by suleimanas** [comment, marked as answer]
it's a webpack 4 requirement, with v14 i have no issues running some old/small projects i have. but not after upgrading to v16

**Comment by chupzzz** [comment]
Hello from 2025 and Quasar v1 user 😅
We have quite complicated project on v1 and no budget for upgrade - so continue to support this legacy.

But we're using latest Node to dev/build Quasar v1!

Just add ```NODE_OPTIONS=--openssl-legacy-provider``` to your ```quasar``` command and it will run with no issues.

Fragment from our package.json:

```json
"scripts": {
    "dev": "NODE_OPTIONS=--openssl-legacy-provider quasar dev",
    "build": "NODE_OPTIONS=--openssl-legacy-provider quasar build"
}
```

Hope it helps someone who got stuck in legacy.
